### PR TITLE
Relay substrate withdrawing transaction

### PR DESCRIPTION
**Pull request by AhmedKorim**
## PR Checklist
Please check if your PR fulfills the following requirements:
Should be merged after https://github.com/webb-tools/relayer/pull/86

- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## What is the current behavior?
Adding withdrawal submission throw relayers for substrate mixer

Issue Number: #129 

## What is the new behavior?
Users will be able to utilize relayers for withdrawing tx

## Does this PR introduce a breaking change?
- [ ] Yes
- [ ] No

### CODE REVIEW

File: packages/mixer/src/components/Withdraw/Withdraw.tsx [hunk 1]
```diff
@@ -129,7 +131,12 @@ export const Withdraw: React.FC<WithdrawProps> = () => {
         }
       });
   };
-
+  const relayers = relayersState.relayers;
+  const activeRelayer = relayersState.activeRelayer;
+  console.log({
+    relayersState,
+    relayers,
+  });
```
Comment (nepoche) [inline packages/mixer/src/components/Withdraw/Withdraw.tsx:139]: Can we do some logging cleanup / leave ones in that are helpful?

File: packages/react-environment/src/webb-context/relayer/webb-relayer.class.ts [hunk 2]
```diff
@@ -116,9 +133,9 @@ export class WebbRelayerBuilder {
           : new Map(),
         substrate: info.substrate
           ? Object.keys(info.substrate)
-              .filter((key) => info.substrate[key]?.account && nameAdapter(key, 'evm') != null)
+              .filter((key) => info.substrate[key]?.beneficiary || nameAdapter(key, 'substrate') != null)
```
Comment (nepoche) [inline packages/react-environment/src/webb-context/relayer/webb-relayer.class.ts:136]: Will the relayer for substrate always return `beneficiary`? Or is it still optional and if a beneficiary is not set in the config, will it return `account`?

File: packages/react-environment/src/webb-context/mixer/mixer-withdraw.ts [hunk 3]
```diff
@@ -63,7 +63,7 @@ export abstract class MixerWithdraw<T> extends EventBus<MixerWithdrawEvents> {
   }
 
   // todo switch to the reactive api
-  get relayers(): Promise<WebbRelayer[]> {
+  get relayers(): Promise<WebbRelayer[]> | WebbRelayer[] {
```
Comment (nepoche) [inline packages/react-environment/src/webb-context/mixer/mixer-withdraw.ts:66]: This seems strange, shouldn't it be one or the other?

**Review by nepoche — CHANGES_REQUESTED** [review]

File: packages/react-environment/src/webb-context/relayer/webb-relayer.class.ts (hunk 2 above)
Comment (AhmedKorim) [reply packages/react-environment/src/webb-context/relayer/webb-relayer.class.ts:136]: I believe the `account` the field is deprecated in favor of the `beneficiary`, And relayer that supports substrate will have the `beneficiary` key  not account that's why I just added just `beneficiary` and it wont hurt adding `account` as well if you think so

File: packages/mixer/src/components/Withdraw/Withdraw.tsx (hunk 1 above)
Comment (AhmedKorim) [reply packages/mixer/src/components/Withdraw/Withdraw.tsx]: removed, will add a cleanup task for console logs from the whole dApp with #243

File: packages/react-environment/src/webb-context/relayer/webb-relayer.class.ts (hunk 2 above)
Comment (nepoche) [reply packages/react-environment/src/webb-context/relayer/webb-relayer.class.ts]: Nah it's better not to support it. Much better idea for it always to be beneficiary. Would you mind just adding a comment above line 127 that account is to support Mainnet Harmony and Shiden relayer deployments?

Comment (AhmedKorim) [reply packages/react-environment/src/webb-context/relayer/webb-relayer.class.ts]: This is why I didn't add it @nepoche! , will restore to what I first implemented and add the comment you suggested

**Pull request merged**